modules/apache-airflow/templates/values.yaml

```diff
@@ -1750,7 +2081,7 @@ pgbouncer:
   command: ["pgbouncer", "-u", "nobody", "/etc/pgbouncer/pgbouncer.ini"]
   # Args to use for PgBouncer(templated).
   args: ~
-  auth_type: md5
+  auth_type: scram-sha-256
```

Comment:
What is the significance of this change? Or is it the change in defaults

Reply:
We are not using their version of pgbouncer so we are not affected. This was the default in the new chart version